fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -35,6 +37,7 @@ export interface LocationSuggestProps
   schemeId: string;
   usageTypeCode?: string;
   initialValue?: string;
+  showBreadcrumbs?: boolean;
```

Comment:
Does this need to be added here: https://github.com/seek-oss/wingman/blob/95d6662691ce3b6cb29783da4a185b560aa50c99/fe/lib/components/LocationSuggest/LocationSuggest.stories.tsx#L21

Reply:
No, it is optional. Storybook will just have a "set boolean" button, after clicking you can set true or false. If it's not set, it'll be the same as false.

I've added it anyway though just to be more explicit with the props